packages/deployments/contracts/contracts/core/connext/libraries/AssetLogic.sol

```diff
@@ -361,7 +373,8 @@ library AssetLogic {
     AppStorage storage s = LibConnextStorage.connextStorage();
 
     // If the adopted asset is the local asset, no need to swap.
-    address adopted = s.canonicalToAdopted[_key];
+    TokenConfig memory config = getConfig(_key);
```

Comment:
Different pattern. Here `memory` is used.

Other places use `storage`:
```solidity
TokenConfig storage config = getConfig(_key);`
```

Reply:
I agree that this should use storage

We might not need every variable from the config, so no reason to SLOAD all of it and copy to mem